README.rst

```diff
@@ -38,6 +38,72 @@ HTML gallery of examples from any set of Python scripts.
    :width: 80%
    :alt: A demo of a gallery generated by Sphinx-Gallery
 
+Quickstart
+==========
+
+Sphinx-Gallery can be used to generate an example gallery for a library, as well as a stand-alone web page showcasing examples of a particular Python package, module, or class.
+
+Let's get started with a simple example or checkout the `documentation <sphinx-gallery.github.io>`_ for introductions on how to use it and more...
+
+Install via ``pip``
+-------------------
+
+.. installation-begin-content
+
+You can do a direct install via ``pip`` by using:
+
+.. code-block:: bash
+
+    $ pip install sphinx-gallery "sphinx>=4.0" pillow
+
+.. warning::
+    Sphinx-Gallery will not manage its dependencies when installing, thus
+    you are required to install them manually. Our minimal dependencies
+    are **Sphinx >= 4** and Pillow, which we use for scaling images.
+
+.. note::
+    Sphinx-Gallery has also support for scraping images from packages like
+    Matplotlib, Seaborn, and PyVista. We recommend installing system ``optipng``
```

Comment:
could we clarify that we support matplotlib and matplotlib based packages such as Seaborn and PyVista?

Reply:
well, this is taken from the actual Readme, so do you have suggestions on how to improve it?

Reply:
Ah good point. Our [image scraper doc](https://sphinx-gallery.github.io/stable/configuration.html#image-scrapers) says:

> The default value is 'image_scrapers': ('matplotlib',) which only scrapes Matplotlib images. Note that this includes any images produced by packages that are based on Matplotlib, for example Seaborn or Yellowbrick.

What about:

"Sphinx-Gallery has also support for scraping images from Matplotlib and Matplotlib based packages such as Seaborn, and PyVista."

Reply:
updated